### DISCUSSION TITLE
Geolocate a Navisworks model

### DISCUSSION THREAD

**Discussion by Mohammed-Benotmane** [iTwin/itwinjs-core#6054] [Q&A]
When I try to geolocate a Navisworks model using a Revit it's always geolocating the model in the same place (Montreal Canada):

![image](https://github.com/iTwin/itwinjs-core/assets/39698080/6f4cc198-5027-4ad6-b000-8b2f738f2f22)

Here are the steps I usually follow:

1- Move the Navisworks model to (0,0,0).
2- Create an empty Revit file and give it the coordinates (lat,lng) of the project.
3- Upload the 2 files to Synchro Control.
4- Create an iModel then a connection (using the Revit as the geolocation file).

I also tried uploading only the Navisworks file, before it was showing the BIM without the background map but now it's geolocating the model into the same position (Montreal Canada).

**Comment by pmconne** [comment]
For assistance with products like Synchro Control, please contact product support.

**Comment by Johannes-repos** [comment]
Hello,
you will need to have a geolocation file for this. You can either use a Civil3D file or a MicroStation file with a global coordinate system attached.
Navisworks as is, doesn't have any GCS information stored. In case your Revit file has X and Y coordinates (shared) set correctly, you could export a DWG with shared coordinates and attach this to C3D. Assign a coordinate system and use that file as a geolocation file.

Please also see here:
https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/68002/using-a-civil3d-geolocation-file-to-publish-revit-projects

Detailed instructions on how to work with NWD files are coming soon
